### DISCUSSION TITLE
Is there a wrapper for the listings-restrictions api?

### DISCUSSION THREAD

**Discussion by jbnels** [saleweaver/python-amazon-sp-api#299] [FAQ] [answered]
I'm trying to get listing restrictions using the [listings-restrictions api](https://github.com/amzn/selling-partner-api-docs/blob/main/guides/en-US/use-case-guides/listings-restrictions-api-use-case-guide/listings-restrictions-api-use-case-guide_2021-08-01.md).  Is this available?

**Comment by saleweaver** [comment, marked as answer]
Hi,

there isn't at the moment. If you look at the make_endpoint dir and file from this repo, you can simply generate the endpoint. 

I'm on the road until Monday, I'll add this endpoint then if you haven't created a PR.
